grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -593,12 +604,34 @@ class OrderService {
                         BigDecimal qtyPerUom = uomParts.length > 1 ? BigDecimal.valueOf(Double.valueOf(uomParts[1])) : null
                         orderItem.quantityUom = uom
                         orderItem.quantityPerUom = qtyPerUom
+                    } else {
+                        throw new IllegalArgumentException("Missing unit of measure.")
```

Comment:
We could potentially default to EA, but I think it's better to err on the side of caution for now.